guide/3.2/plugin-fabric.md

```diff
@@ -131,7 +131,76 @@ node_templates:
               important_prop2: 300
 {%endhighlight%}
 
-This example is very similar to the previous one with the following difference. If the fabric task you want to execute is already installed in the python environment in which the operation will run, you can specify the python path to this function.
+This example is very similar to the previous one with the following difference. If the fabric task you want to execute is already installed in the python environment in which the operation will run, you can 
+specify the python path to this function.
+
+
+# Running scripts
+
+Fabric plugin can execute scripts remotely and provides access to the `ctx` API for interacting with Cloudify in the same manner [script-plugin](plugin-script.html) does.
+
+Example:
+
+{% highlight yaml %}
+node_templates:
+  example_node:
+    type: cloudify.nodes.WebServer
+    interfaces:
+      cloudify.interfaces.lifecycle:
+        start:
+          implementation: fabric.fabric_plugin.tasks.run_script
+          inputs:
+            # Path to the script relative to the blueprint directory
+            script_path: scripts/start.sh
+            MY_ENV_VAR: some-value
+{%endhighlight%}
+
+{%note title=Note%}
+Fabric plugin doesn't support evaluating Python scripts as script-plugin does and therefore the `ctx` object cannot be used in Python scripts as script-plugin allows.
```

Comment:
Fabric plugin -> The fabric plugin
as script-plugin -> as the script-plugin
as script-plugin allows -> as the script-plugin